litho-core/src/main/java/com/facebook/litho/ComponentTree.java

```diff
@@ -169,6 +175,13 @@ public void run() {
   @GuardedBy("mCurrentCalculateLayoutRunnableLock")
   private @Nullable CalculateLayoutRunnable mCurrentCalculateLayoutRunnable;
 
+  private final boolean mSingleLayout;
```

Comment:
This doesn't communicate a lot about what it'll do. Can we call it something else?

Reply:
Yeah I was struggling to find a better name, open to suggestions. I'll try to come up with something.